locale/en-US/standalone.properties

```diff
@@ -61,3 +61,5 @@ rooms_read_while_wait_offer=Want something to read while you wait?
 ## replaced by the brand name.
 standalone_title_with_room_name={{roomName}} — {{clientShortname}}
 status_error=Something went wrong
+
+clientSuperShortname=Hello
```

Comment:
Please add it like this:

## LOCALIZATION NOTE(clientSuperShortname): This should not be localized and
## should remain "Hello" for all locales.
clientSuperShortname=Hello